### PR TITLE
{bio}[foss/2018b] MACS2 v2.2.5 w/ Python 3.6.6

### PR DESCRIPTION
**Pull request by arielzn**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/MACS2/MACS2-2.2.5-foss-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+name = 'MACS2'
+version = '2.2.5'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/taoliu/MACS/'
+description = "Model Based Analysis for ChIP-Seq data"
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/taoliu/MACS/archive']
```
Comment (boegel) [inline easybuild/easyconfigs/m/MACS2/MACS2-2.2.5-foss-2018b-Python-3.6.6.eb]: Any reason for now downloading from PyPI like the existing `MACS` easyconfigs do?

```python
source_urls = [PYPI_SOURCE]
sources = [SOURCE_TAR_GZ]

Comment (arielzn) [reply easybuild/easyconfigs/m/MACS2/MACS2-2.2.5-foss-2018b-Python-3.6.6.eb]: according to the changelog on latest release for 2.2.5

https://github.com/taoliu/MACS/releases

They are including only on the github release some extra things for testing purposes.

But if these tests are not being triggered it's true we could stay with the PyPI one.

Comment (verdurin) [reply easybuild/easyconfigs/m/MACS2/MACS2-2.2.5-foss-2018b-Python-3.6.6.eb]: Any update on this one?

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/f6f48ba104807c4c896dc61c03e599ae for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @arielzn!

**Pull request merged**